docs/extending/custom_bulk_actions.md

```diff
@@ -30,7 +30,7 @@ The attributes are as follows:
 
 -   `display_name` - The label that will be displayed on the button in the user interface
 -   `aria_label` - The `aria-label` attribute that will be applied to the button in the user interface
--   `action_type` - A unique identifier for the action. Will be required in the url for bulk actions
+-   `action_type` - A unique identifier for the action (required in the url for bulk actions)
```

Comment:
```suggestion
-   `action_type` - A unique identifier for the action (required in the URL for bulk actions)
```

Let's update this to use the proper acronym URL while we are at it. I will just commit this suggestion.